tests/python/gstaichi/lang/fast_caching/test_args_hasher.py

```diff
@@ -253,3 +254,33 @@ class MyConfigChecked:
     assert h_base is not None
     assert h_base == h([same], [None])
     assert h_base != h([diff], [None])
+
+
+@test_utils.test()
+def test_args_hasher_torch_tensor() -> None:
+    seen = set()
+    arg = torch.zeros((2, 3), dtype=float)
+    for it in [0, 1]:
```

Comment:
As same as `test_args_hasher_custom_torch_tensor`, using `range(2)` would be better? But no big deal.